src/entities/users.entity.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  CreateDateColumn,
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+  BaseEntity,
+} from 'typeorm';
+
+export enum UserRole {
+  Admin = 'Admin',
+  Student = 'Student',
+  HR = 'HR',
+}
+
+@Entity()
+export class UsersEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({
+    unique: true,
+  })
+  email: string;
+
+  @Column({
+    default: false,
+  })
+  isActive: boolean;
+
+  @Column({
+    type: 'enum',
+    enum: UserRole,
+  })
+  role: UserRole;
+
+  @Column()
+  pwd: string;
+
+  @Column({
+    generated: 'uuid',
+    unique: true,
+  })
+  registerToken: string;
+
+  @Column({
+    nullable: true,
+    default: null,
+  })
+  currentTokenId: string | null;
+
+  @CreateDateColumn({
+    type: 'timestamp',
+  })
+  createdAt: Date;
```

Comment:
Nie wiem do czego będzie potrzebna ta kolumna, ale jeśli potrzebna będzie data zapisania tego rekordu, to timestamp będzie problematyczny, bo on aktualizuje automatycznie datę przy każdym updacie rekordu.